### DISCUSSION TITLE
Requesting ability to create custom group-by within the Kanban rows

### DISCUSSION THREAD

**Discussion by mohan-the-octocat** [community/community#5669] [Projects and Issues] [answered]
Happy to see the project view shaping up. 

I was trying to build a Kanban view to track our team's activities. While the status field is evident from the columns, it would be helpful to 

* Have column views based on other fields - Assignee, Customer etc
* Have ability to group the issues in each column by other fields   - Customer, due-by-date etc

Our current dashboard looks like this...
![image](https://user-images.githubusercontent.com/83639549/133548588-0c97b456-90b8-44c8-9a0d-d1b328e78281.png)

We'd ideally want to show this grouped by the assignee or other fields :).

**Comment by erichexter** [comment, marked as answer]
So I would expect that the group by would be swimlanes (rows) rather then columns.

**Reply by jesfer** [reply]
That is how I envision it as well. But I wonder if it would be useful to be able to group different columns by different fields... can't really think of a good use case but maybe I'm stuck in my ways of thinking...

**Comment by jesfer** [comment]
Columns based on other fields would be awesome.

**Comment by mohan-the-octocat** [comment]
Thanks for the call out folks! Was meaning to ask for group by rows! Modified the title and the description to reflect the same!
